pytorch_lightning/trainer/__init__.py

```diff
@@ -595,12 +595,11 @@ def on_train_end(self):
 print_nan_grads
 ^^^^^^^^^^^^^^^
 
-Prints gradients with nan values
+.. warning:: .. deprecated:: 0.7.2.
```

Comment:
there are many other deprecation warnings which have only `.. warning::` and missing the next `.. deprecated::` so shall it be also changed? there ate usually in inits and deprecated loggers...

Reply:
Hi, saw this comment too late. Yeah good idea, let's do it!